roles/mesos-agent/tasks/main.yml

```diff
@@ -15,13 +18,13 @@
     - { file: '/etc/mesos-slave/hostname', content: "{{mesos_hostname}}" }
     - { file: '/etc/mesos-slave/containerizers', content: 'docker,mesos' }
     - { file: '/etc/mesos-slave/isolation', content: 'cgroups/cpu,cgroups/mem' }
-    - { file: '/etc/mesos-slave/executor_registration_timeout', content: '5mins' }
+    - { file: '/etc/mesos-slave/executor_registration_timeout', content: '2mins' }
```

Comment:
Usually takes less than a minute, but sometimes connection fails. I will revert, if we ever see timeout on slow connections